packages/mobile/src/components/dropdown/dropdown.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { ReactElement } from 'react';
+import DropDownPicker from 'react-native-dropdown-picker';
+import {
+  FormControl,
+  FormControlPath,
+  FormControlValues,
+} from '~/common/types/types';
+import { useState, useCustomTheme, useFormControl } from '~/hooks/hooks';
+import { Text, View } from '~/components/components';
+import { globalStyles } from '~/styles/styles';
+import { styles } from './styles';
+
+type Props<T extends FormControlValues> = {
+  label: string;
+  name: FormControlPath<T>;
+  control: FormControl<T>;
+  items: {
+    label: string;
+    value: string;
+  }[];
```

Comment:
Just for readability's sake
```suggestion
  items: Array<{
    label: string;
    value: string;
  }>;
```

Reply:
corrected